utils/api/slack.js

```diff
@@ -55,9 +55,9 @@ async function sendSlackMessage(webHookURL, opsIncident, user) {
     attachments: attachments,
   }
 
-  ux.spinner.start('Posting to Slack')
+  await ux.spinner.start('Posting to Slack')
   await webhook.send(slackMessage)
-  ux.spinner.stop('Done!')
+  await ux.spinner.stop('Done!')
```

Comment:
Let's be more descriptive for these success messages. Here maybe it could be `Posted to Slack!` with a green check mark emoji at the start of the string.